### DISCUSSION TITLE
How do you get the values from the select element?

### DISCUSSION THREAD

**Discussion by raulgonzalezdev** [csandman/chakra-react-select#97] [Q&A] [answered]
**My question is...**
Hello, First let me congratulate you for such a great job with this control, I would like to know how I get the list of values of the selected elements

**Comment by csandman** [comment, marked as answer]
Thanks for your feedback!  For the basics of how to use this component, you should check out the docs for the original react-select component: https://react-select.com/home

But I can also give you a simple example of how to do so, you just need to use the `value` and `onChange` props to manually keep track of the value using state:

```js
import { useState } from "react";
import { Select } from "chakra-react-select";

export const flavors = [
  { value: "vanilla", label: "Vanilla" },
  { value: "chocolate", label: "Chocolate" },
  { value: "strawberry", label: "Strawberry" },
  { value: "salted-caramel", label: "Salted Caramel" }
];

const Example = () => {
  const [selectedOptions, setSelectedOptions] = useState([]);

  return (
    <Select
      isMulti
      name="flavors"
      options={flavors}
      placeholder="Select some flavors..."
      // Add these props to the component
      value={selectedOptions}
      onChange={setSelectedOptions}
    />
  );
};
```

**Comment by raulgonzalezdev** [comment]
thanks, great resposnse
